docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM --platform=$BUILDPLATFORM alpine:3.18 as buildenv
+
+RUN apk add curl 
+
+ENV HOME /home/builder
+
+RUN curl -fsSL https://raw.githubusercontent.com/pulumi/customer-managed-deployment-agent/main/install.sh | sh
+
+FROM alpine:3.18
```

Comment:
Do you need alpine, or would the scratch image work?

Reply:
I believe I need alpine, but cannot remember why, but have not tested. For now I would like to just go with this and can simplify later if there is demand.